clients/privacy-center/public/fides-js-components-demo.html

```diff
@@ -312,7 +312,11 @@ <h2>Property id</h2>
         console.log("Fides has been initialized!");
         // Pretty-print the fides consent object and add it to the page.
         document.getElementById("consent-json").textContent = JSON.stringify(
-          Fides.consent,
+          Fides.experience &&
+            Fides.experience.experience_config &&
+            Fides.experience.experience_config.component === "tcf_overlay"
+            ? Fides.tcf_consent
+            : Fides.consent,
```

Comment:
This stuff might not be needed. It relates to my question on our call about what the `tcf_consent` string is. Doesn't hurt. But probably doesn't help either.